source/server.ts

```diff
@@ -1,12 +1,12 @@
 import type { NextHandleFunction } from 'connect';
-import Fibers from 'fibers';
-import YAML from 'js-yaml';
+import { current } from 'fibers';
```

Comment:
This is not correct, as `current` get's overridden in the runtime, therefore we have to access it every time.